wizard/src/types.ts

```diff
@@ -91,8 +91,14 @@ export interface CsvData {
 
 // Types used for Detection.
 
-// DetectedFormat denotes the format type associated with some detected type..
-export interface DetectedFormat {
+// The type of place, e.g. country.
+export interface PlaceType {
+  typeName: string;
+  displayName: string;
+}
+
+// The Data Commons property of a place, e.g. longitude.
+export interface PlaceProperty {
   propertyName: string;
   displayName: string;
 }
```

Comment:
Can we generalize PlaceType and PlaceProperty so that this can be:

```
Entity {
  dcName: string;
  displayName: string;
}

export interface TypeProperty {
  type: Entity;
  property?: Entity;
}
```

Reply:
Done. With a slightly minor edit where I am also using PlaceType and PlaceProperty as extensions of Entity